content/modules/add-module-to-database.md

```diff
@@ -43,3 +43,84 @@ To add modules to a database:
 
 1. Click **Show advanced options** and enter a port number for the database, such as: **12543**.
 1. Click **Activate**.
+
+
+## Upgrading the module for the database
```

Comment:
Can this only be done from the terminal with `rladmin` or is there an option to do this from the API and/or admin console?